app/_hub/tibbycat/jq-transformer/index.md

```diff
@@ -0,0 +1,155 @@
+  * Transform the request & response to simplify the interface to your end service (eg Confluent's kafka REST proxy)
+  * Remove sensitive(/private) information from your response payload.
+  * Reduce your response payload size.
+  NB: Content-Type header will probably affect your results. (application/.*json is probably the safest)
+
+#support_url:
+  # (Optional) A specific URL of your own for this extension.
+  # Defaults to the url setting in your publisher profile.
+
+source_url:
+  https://github.com/tibbycat/kong-jq-transformer
+
+license_type:
+  https://opensource.org/licenses/alphabetical
+
+#license_url:
+  # (Optional) Link to your custom license.
+
+#privacy_policy:
+  # (Optional) If you have a custom privacy policy, place it here
+
+#privacy_policy_url:
+  # (Optional) Link to a remote privacy policy
+
+#terms_of_service:
+  # (Optional) Text describing your terms of service.
+
+#terms_of_service_url:
+  # (Optional) Link to your online TOS.
+
+# COMPATIBILITY
+# In the following sections, list Kong versions as array items.
+# Versions are categorized by Kong edition and their known compatibility.
+# Unlisted Kong versions will be considered incompatible.
+# Uncomment at least one of 'community_edition' or 'enterprise_edition'.
+# Add array-formatted lists of versions under their appropriate subsection.
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 0.12.x
+      - 0.14.x
+#    incompatible:
+  enterprise_edition:
+    compatible:
+      - 1.3.x
+      - 1.5.x
+      - 2.1.x
+#    incompatible:
+
+
+# EXAMPLE kong_version_compatibility block: this example shows how to indicate
+# various compatibilities. Also see other extension files in _app/_hub/ for more examples.
+#kong_version_compatibility:
+#  community_edition:
+#    compatible:
+#      - 0.12.x
+#      - 0.13.x
+#      - 0.14.x
+#  enterprise_edition:
+#    compatible:
+#      - 0.32-x
+#      - 0.33-x
+#      - 0.34-x
+
+#########################
+# PLUGIN-ONLY SETTINGS below this line
+# If your plugin is an extension of the core platform, ALL of the following
+# lines must be completed.
+# If NOT defined as a 'plugin' in line 32, delete all lines up to '# BEGIN MARKDOWN CONTENT'
+
+params: # Metadata about your plugin
+  name: jq-transformer
+  service_id:
+    # Boolean - whether this plugin can be applied to a Service.
+    # Affects generation of examples and config table.
+  consumer_id:
+    # Boolean - whether this plugin can be applied to a Consumer.
+    # Affects generation of examples and config table.
+  route_id:
+    # whether this plugin can be applied to a Route.
+    # Affects generation of examples and config table.
+  protocols:
+    # List of protocols this plugin is compatible with, in array format.
+    # Valid values: "http", "https", "tcp", "tls"
+    ["http", "https"]
+  dbless_compatible:
```

Comment:
IS this plugin compatible with db-less mode? k8? Please see tabs for OSS and Enterprise here https://docs.konghq.com/hub/plugins/compatibility/

Reply:
I am unsure of this as I don’t use dB-less mode. However, I don’t explicitly interact with the dB. And commonly use decK to apply the plugin.

Reply:
Would it be acceptable to say ‘partially’ as this ‘could be’ but is ‘untested’?

Reply:
Probably not but I'll defer to @eskibars